b9lab-content/2-what-is-cosmos/02-blockchain-and-cosmos.md

```diff
@@ -0,0 +1,154 @@
+
+*How is Cosmos an internet of blockchains?* Cosmos is a **network of interoperable blockchains**, each implemented with different properties suitable for their individual use cases. Cosmos lets developers create blockchains that maintain sovereignty, free from any "main chain" governance, that allow for fast transaction processing, and that are interoperable. With this, a great variety of use cases become possible.
+
+To achieve this vision and type of network, the ecosystem relies on an **open source toolkit**, including the [Inter-Blockchain Communication (IBC)](https://ibcprotocol.org/) protocol, its implementation in the [Cosmos SDK](https://v1.cosmos.network/sdk), and [Tendermint](https://tendermint.com/) as the base layer with state finality. The toolkit, a set of modular, adaptable, and interchangeable tools, helps not only to quickly spin up a blockchain, but also facilitates the customization of secure and scalable chains.
+
+Let's now look at the problems Cosmos solves.
+
+### How Does Cosmos Solve the Scalability Issue?
+
+As mentioned previously, scalability is a big issue area when it comes to blockchain technology. Cosmos allows applications to scale to millions of users.
+
+Cosmos addresses **two types of scalability**:
+
+* **Horizontal scalability**: Scaling by adding similar machines to the network. When scaling out, horizontally, the network can accept more nodes to participate in the state replication, consensus observation, and any activity that queries the state.
+* **Vertical scalability**: Scaling by improving the network's components so as to increase its computational power. When scaling up, vertically, the network can accept more transactions, and any activity that modifies the state.
+
+In a blockchain context, vertical scalability is typically achieved through the optimization of the consensus mechanism and applications running on the chain. On the consensus side, Cosmos achieves vertical scalability with the help of the Tendermint BFT. For instance, the Cosmos Hub currently conducts transactions in seven seconds. In a single blockchain context, the only remaining bottleneck is then the application.
+
+Consensus mechanism and application optimization of one's blockchain can only take one so far. To overcome the limits of vertical scalability, Cosmos' multi-chain architecture allows for **one application to run in parallel** on different chains, all the while being operated by the same validator set. This inter-chain horizontal scalability theoretically allows for infinite vertical-like scalability.
+
+<HighlightBox type="info">
+
+A **validator** is one or more cooperating computers that participate in the consensus by, among other things, creating blocks.
+
+</HighlightBox>
+
+### How Does Cosmos Promote Sovereignty?
+
```

Comment:
I think we could say more here about the application specific blockchains. It's not just that its 1 layer governance, but that the chain can be tailored to the needs of the application in many ways, and can define the level of security it needs (and needs to pay for).

Also on the 2 layer gov front the base chain can upgrade in ways that break your applications. And its base economics can interfere with your own economics. Etc. So building out your own chain gives you a full stack environment with users, validators, community, infrastructure explicitly dedicated to that application.

Reply:
It might be worth linking to some good video presentations where we go into this kinda stuff more. I think Sunny's Many Chains One Ecosystem might be good